notifications/notification.go

```diff
@@ -129,50 +135,66 @@ func NotifyPrompt(id, msg string, actions ...Action) *Notification {
 	return notify(Prompt, id, msg, actions...)
 }
 
-func notify(nType Type, id string, msg string, actions ...Action) *Notification {
+func notify(nType Type, id, msg string, actions ...Action) *Notification {
 	acts := make([]*Action, len(actions))
 	for idx := range actions {
 		a := actions[idx]
 		acts[idx] = &a
 	}
 
-	if id == "" {
-		id = utils.DerivedInstanceUUID(msg).String()
-	}
-
-	n := Notification{
+	return Notify(&Notification{
 		EventID:          id,
-		Message:          msg,
 		Type:             nType,
+		Message:          msg,
 		AvailableActions: acts,
+	})
+}
+
+// Notify sends the given notification.
+func Notify(n *Notification) *Notification {
+	// Derive missing information.
+	if n.Message == "" {
+		n.Title = n.Message
+	}
```

Comment:
I guess you meant `Message = Title`. Though I'd rather not do that because we might end up showing the notification title twice in the UI.